examples/fscloud/main.tf

```diff
@@ -10,6 +10,18 @@ module "resource_group" {
   existing_resource_group_name = var.resource_group
 }
 
+##############################################################################
+# Key Protect Instance
+##############################################################################
+
+module "key_protect_module" {
+  source            = "terraform-ibm-modules/key-protect/ibm"
+  version           = "2.2.1"
+  key_protect_name  = "${var.prefix}-key-protect-instance"
+  resource_group_id = module.resource_group.resource_group_id
+  region            = "us-south"
```

Comment:
use `var.prefix`

Reply:
@Ak-sky this is still outstanding

Reply:
in key_protect_name should I use `var.prefix` only?